signac/common/config.py

```diff
@@ -38,12 +56,14 @@ def search_tree(root=None):
 
 
 def search_standard_dirs():
+    """Locates signac configuration files in standard directories."""
```

Comment:
```suggestion
    """Locates signac configuration files in standard directories.
    
    """
```

[One line summary](https://numpydoc.readthedocs.io/en/latest/format.html#sections) section shows that the inverted commas should be in a different line. (Though I'm not sure)

Reply:
Interesting. I hadn't noticed that in the examples before. I think that we're pretty consistent with using a single line here (no blank lines), and `pydocstyle` doesn't seem to complain, so I think I would prefer to leave this as it is.